containerbuddy/onstart.sh

```diff
@@ -2,24 +2,61 @@
 
 # Refresh all config files in order
 #
-# This script is typically called once at the container start, but 
+# This script is typically called once at the container start, but
 # it can be called manually if WP config details must be changed
 
-# The database and memcached config files are separate to avoid collisions 
+# The database and memcached config files are separate to avoid collisions
 # if their backends' onChange handlers are triggered simultaneously
-/opt/containerbuddy/reload-db.sh
-/opt/containerbuddy/reload-memcached.sh
+echo "******running onstart script*********"
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep mysql-primary`  ]]
+do
+  echo "mysql-primary not healthly...."
+  sleep 5
+done
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
```

Comment:
You want to use `$CONSUL_HOST/v1/health/service/$SERVICE_NAME?passing` here? See also https://github.com/autopilotpattern/wordpress/pull/3#discussion_r58459297